psydac/api/ast/nodes.py

```diff
@@ -153,7 +157,7 @@ def stop(self):
     def length(self):
         return self._length
 
-    def set_range(self, start=Tuple(*[S.Zero]*3), stop=None, length=None):
+    def set_range(self, start=TensorInteger(1), stop=None, length=None):
```

Comment:
Why do you use `1` as default value instead of `0`?

Reply:
ah yes, sorry

Reply:
that's why the test is not passing